docs/release-notes.md

```diff
@@ -25,6 +25,8 @@ nav_order: 9
 
 ### Docs changes
 
+- Document that `hash` fields describe decompressed contents/config
```

Comment:
Maybe:
```suggestion
- Document that `hash` fields describe decompressed data
```

Reply:
Ah, ok, and added the change for the `certificate_authorities`